overlord/devicestate/devicestate.go

```diff
@@ -637,3 +637,48 @@ func Remodeling(st *state.State) bool {
 	}
 	return false
 }
+
+type recoverySystemSetup struct {
+	// Label of the recovery system
+	Label string `json:"label"`
+	// Directory inside the seed filesystem where the recovery system files
+	// are kept
```

Comment:
can you make an example?  from this description alone I'm not sure what is this directory, maybe its clear in the code, but not here

Reply:
Updated the comments to reflect where/when each field is set.